### DISCUSSION TITLE
Express-graphql is deprecated + Apollo

### DISCUSSION THREAD

**Discussion by Bibazavr** [payloadcms/payload#2229] [Feature Requests & Ideas]
Hello!

[Express-graphql is deprecated](https://github.com/graphql/express-graphql#this-library-is-deprecated)

Payload cms going to [graphql-http](https://github.com/graphql/graphql-http)?

And it is possible to use Apollo graphQL with Payload cms?

**Comment by jnovak-SM2Dev** [comment]
Any update on this? I'd be nice to be able to use apollo studio to test graphql queries.
